src/external/time.rs

```diff
@@ -1,5 +1,6 @@
 use super::Result;
```

Comment:
I think it would be nicer if this were in `src/util/date.rs`, since it directly relates to dates.
Or is it only concerned with mint dates? Then I think it should be in `src/external/mint.rs`